src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -779,6 +788,63 @@ void TableDockWidget::exportGroupsToSpreadsheet() {
     }
 }
 
+
+
+
+void TableDockWidget::exportGroupsToSpreadsheet_polly() {
```

Comment:
As far as I can see, I don't think it's a required function, instead **exportGroupsToSpreadsheet** can be used!

Reply:
No..This one is quite different..